### DISCUSSION TITLE
关于服务器组和nameserver的疑问

### DISCUSSION THREAD

**Discussion by iIl1o0O304** [pymumu/smartdns#1473] [Q&A]
1. 如图，一个上游服务器可以属于多个服务器组吗？
![1](https://github.com/pymumu/smartdns/assets/128894450/87f4acac-32ab-4ae0-869a-e7f31ea5a2e5)
2. 如图，一个nameserver规则可以对应多个服务器组吗？
![2](https://github.com/pymumu/smartdns/assets/128894450/c920dbc5-c27c-4ccb-ba59-85c179fe7580)

**Comment by PikuZheng** [comment]
不能这么写（逗号会识别为名称的一部分
实测不生效（同一行有两个组的，只有后一个生效

**Comment by pymumu** [comment]
server选项能支持多个group，但luci只能配置一个，如果要配置多个，可以在附加选项再写-group xxxgroup

nameserver选项仅支持配置一个group
